jetty-servlets/src/main/java/org/eclipse/jetty/servlets/CrossOriginFilter.java

```diff
@@ -501,11 +511,41 @@ private String commify(List<String> strings)
         return builder.toString();
     }
 
+    /** 
+     * creates a pattern to match any of a list of  string
+     * If the input list is ["foo, "bar"] it will generate
+     * a Pattern for the regex "(\\Efoo\\E|\\Ebar\\E) 
```

Comment:
The first and third `\\E` should be `\\Q`.

Reply:
Ops! thanks, fixed at c25f9511